openedx/features/learner_analytics/views.py

```diff
@@ -51,11 +63,12 @@ def get(self, request, course_id):
             'uses_pattern_library': True,
             'grading_policy': course.grading_policy,
             'assignment_grades': self.get_grade_data(request.user, course_key),
-            'assignment_schedule': self.get_schedule(request, course_key)
+            'assignment_schedule': self.get_schedule(request, course_key),
+            'weekly_active_users': self.get_weekly_course_activities(course_key),
+            'day_streak': LearnerAnalyticsView.consecutive_active_days_for_user(request.user.id, course_key)
```

Comment:
@dianakhuang this should be `request.user.username` right?

Reply:
Yes. It does look that way.